map/page.js

```diff
@@ -138,10 +203,19 @@ function updateMap(data) {
     " in the Creator Panel.");
     return;
   }
-  const tiles = L.tileLayer('//server.arcgisonline.com/ArcGIS/rest/services/NatGeo_World_Map/MapServer/tile/{z}/{y}/{x}', {
-    maxZoom: 18,
-    attribution: 'Tiles &copy; Esri &mdash; National Geographic, Esri, DeLorme, NAVTEQ, UNEP-WCMC, USGS, NASA, ESA, METI, NRCAN, GEBCO, NOAA, iPC'
+
+
+// Map tile source:
+//    https://leaflet-extras.github.io/leaflet-providers/preview/
+//    Old source was natgeo world map, but that only has data up to zoom 16
```

Comment:
Looks better. If it were easier to add options, I'd add an option here, since existing map users might be used to the existing map style. Hmm, I think we can go ahead and make the change, given that grist-widget is firmly at pre-release version numbers. I tagged the current state of the repo and if anyone doesn't like our change I think https://cdn.jsdelivr.net/gh/gristlabs/grist-widget@v0.0.2/map/index.html would work for them and give us time to add configuration.

Reply:
(Update, no, jsdelivr doesn't work, html is served as plain text. Would need to push v0.0.2 to grist-static.com or someplace.)